test/functional/api/v2/deployments_controller_test.rb

```diff
@@ -4,13 +4,27 @@ class ForemanDeployments::Api::V2::DeploymentsControllerTest < ActionController:
   class FakeTask < ForemanDeployments::Tasks::BaseDefinition
   end
 
+  setup do
+    @stack = ForemanDeployments::Stack.create!(
+      :name => 'stack1',
+      :definition => [
+        'Task1: !task:FakeTask',
+        '  param1: hardcoded',
+        'Task2: !task:FakeTask'
+      ].join("\n"),
+      :organizations => [taxonomies(:organization1), taxonomies(:organization2)],
```

Comment:
IMHO it's about a time to begin using FactoryGirl - the initialization starts being too complicated.